### ISSUE TITLE
How can the community help to maintain the chart ?

### ISSUE THREAD

**Issue by mtparet** [mastodon/chart#83]
Hello,

I see there are CVE not fixed to the helm chart which should be important for most users of the helm chart. https://github.com/mastodon/mastodon/releases/tag/v4.1.5 

How can people (and me) help to maintain the helm chart and get the PRs merged ?

Best,

**Comment by iavael** [comment]
I've raised similar question here but no one replied :\
https://github.com/mastodon/mastodon/discussions/25789

**Comment by timetinytim** [comment]
Since this is a pretty small team, there unfortunately wasn't enough manpower to give a lot of dedicated attention to the helm chart. I was recently brought on to help with devops/infra needs, as well as maintaining stuff like the helm chart, so there will be much more engagement and active development. Sincere apologies for all the silence and delays!

As far as contributing, I have recently been going through a lot of open PRs and issues to both merge and address all of them. Please feel free to both open issues and PRs for suggested changes, but also please note that we plan on doing a rewrite of the chart to address several issues with it. You can see the pinned issue for more details: #129.

The short version is, development will soon shift to the new chart, so any PRs and issues should be moved there once it's done 🙂
